atom/browser/atom_browser_main_parts.h

```diff
@@ -65,6 +72,9 @@ class AtomBrowserMainParts : public brightray::BrowserMainParts {
   // List of callbacks should be executed before destroying JS env.
   std::list<base::Closure> destruction_callbacks_;
 
+  // partition_id => browser_context
+  std::map<std::string, content::BrowserContext*> browser_context_map_;
```

Comment:
There is a `base::ScopedPtrHashMap` for storing pointers, using map to manage pointers is discouraged.